issue707: returning null instead of empty json object

anvk opened the pull request:
for empty XML nodes

I'm not sure how to write tests for this pull request so I would appreciate any help or suggestions on how to do so.

jsdevel commented:
See `request-response-samples`.

jsdevel commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "soap",
-  "version": "0.9.3",
+  "version": "0.9.4",
```
we'll handle version bumping.  Please remove this.

anvk commented:
I removed version bump per request and changed some of the tests to accommodate that change. Although I have problems with tests of such format:

```
<?xml version="1.0" encoding="UTF-8"?>
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
   <soap:Header />
   <soap:Body>
      <Response xmlns="http://www.example.com/v1" />
   </soap:Body>
</soap:Envelope>
```

What would be the result of this? I assume maybe `{}` would make more sense than `null` but again not entirely sure....

jsdevel commented:
see the other response samples under `test/`

jsdevel commented:
@anvk can you update the tests?

jsdevel commented:
Taking this up in https://github.com/vpulim/node-soap/pull/784.
